quartz/components/Breadcrumbs.tsx

```diff
@@ -68,39 +71,40 @@ export default ((opts?: Partial<BreadcrumbOptions>) => {
     const firstEntry = formatCrumb(capitalize(options.rootName), fileData.slug!, "/" as SimpleSlug)
     const crumbs: CrumbData[] = [firstEntry]
 
-    // Get parts of filePath (every folder)
-    const parts = fileData.filePath?.split("/")?.splice(1)
-    if (parts) {
+    // Split slug into hierarchy/parts
+    const slugParts = fileData.slug?.split("/")
+    if (slugParts) {
       // full path until current part
-      let current = ""
-      for (let i = 0; i < parts.length - 1; i++) {
-        const folderName = parts[i]
-        let currentTitle = folderName
+      let currentPath = ""
+      for (let i = 0; i < slugParts.length - 1; i++) {
+        let currentTitle = slugParts[i]
 
         // TODO: performance optimizations/memoizing
         // Try to resolve frontmatter folder title
         if (options?.resolveFrontmatterTitle) {
           // try to find file for current path
-          const currentFile = findCurrentFile(allFiles, folderName)
+          const currentFile = findCurrentFile(allFiles, currentTitle)
           if (currentFile) {
             currentTitle = currentFile.frontmatter!.title
           }
         }
-        // Add current path to full path
-        current += folderName + "/"
+        // Add current slug to full path
+        currentPath += slugParts[i] + "/"
 
         // Format and add current crumb
-        const crumb = formatCrumb(capitalize(currentTitle), fileData.slug!, current as SimpleSlug)
+        const crumb = formatCrumb(
+          capitalize(currentTitle),
```

Comment:
thought it'd look better and more consistent this way but fair enough, fixed in 12d5dd9

Reply:
could we just match explorer behaviour here? just default to the slug if there's no title for it?

Reply:
wdym default to the slug? currently, it is using the full slug of the current file, splitting it by `/` and using those parts to construct breadcrumbs. you can't really directly mirror the explorer behavior here, since it constructs a tree, where you can directly access `node.frontmatter.title`, which you can't as easily do here. there is an option with `resolveFrontmatter: true`, which is just more computationally expensive than just splitting the slug